thrift/from_wire.go

```diff
@@ -123,6 +123,15 @@ func valueFromWireMap(spec *compile.MapSpec, w wire.Map) (map[string]interface{}
 	return result, nil
 }
 
+func mapEnumValueToName(enumSpec *compile.EnumSpec, result int32) interface{} {
+	for _, item := range enumSpec.Items {
```

Comment:
Should we just setup a static mapping so we don't need to iterate like this every time?